pkgs/top-level/perl-packages.nix

```diff
@@ -10400,6 +10400,9 @@ let
     };
     propagatedBuildInputs = [ ArchiveZip DBFile FileWhich IOString ImageSize JSONXS LWP ParseRecDescent TextUnidecode URI XMLLibXML XMLLibXSLT shortenPerlShebang ];
     doCheck = false;  # epub test fails
+    postPatch = ''
+      patchShebangs --host bin
```

Comment:
Is `--host` necessary?

```suggestion
      patchShebangs bin
```

Reply:
Apparently not, it does work as intended without `--host` (I copied the line from another package without paying too much attention).